README.md

```diff
@@ -0,0 +1,46 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is my first project in Microverse
```

Comment:
Kindly, I suggest you write a summary of your project. stating that it is your first project is not as relevant to a reviewer in the future